Eastern-Research-Group/mywaterway
# Feature/new tab disclaimers

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3361854

## Main Changes:
* For background - we are required to display a disclaimer next to all links that open in a new tab. Kiki has agreed with the format shown in the breeze ticket above.

## Steps To Test:
1. Check all pages across the app for links that open in a new tab `target="_blank"` and check that they have a new tab disclaimer either near them or on the page.
